UI/CMakeLists.txt

```diff
@@ -444,6 +471,25 @@ elseif(OS_POSIX)
   if(OS_FREEBSD)
     target_link_libraries(obs PRIVATE procstat)
   endif()
+
+  if(OS_LINUX AND ENABLE_WHATSNEW)
+    find_package(MbedTLS)
+    if(NOT MBEDTLS_FOUND)
+      obs_status(
+        FATAL_ERROR
+        "mbedTLS not found, but required for WhatsNew support on Linux")
+    else()
```

Comment:
Triggering a message with `FATAL_ERROR` acts like a `throw` in CMake - the `else` is not strictly necessary.

Reply:
This one must've fallen through the cracks during some refactoring, I removed the else now.